backend/pkg/api/types/user.go

```diff
@@ -3,6 +3,7 @@ package types
 type UserInfo struct {
 	Id            uint64             `json:"id"`
 	Email         string             `json:"email"`
+	Confirmed     bool               `json:"email_confirmed"`
```

Comment:
Minor: IMO struct fields should have the same name as their respective JSON tags
```suggestion
	EmailConfirmed     bool               `json:"email_confirmed"`
```